lib/widgets/message_list.dart

```diff
@@ -474,11 +474,11 @@ class _UnreadMarker extends StatelessWidget {
   final bool isRead;
   final Widget child;
 
-  // The color hsl(227deg 78% 59%) comes from the Figma mockup at:
+  // The color comes from the Figma mockup at:
   //   https://www.figma.com/file/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=132-9684
   // See discussion about design at:
   //   https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/flutter.3A.20unread.20marker/near/1658008
-  static final color = const HSLColor.fromAHSL(1, 227, 0.78, 0.59).toColor();
+  static const color = Color(0xff466ae8);
```

Comment:
A nice thing about statics is that their initializers only run once. So thankfully this isn't actually hot, and we can leave the source code in the HSL form which is somewhat more transparent to understand.

Reply:
OTOH if this did appear inside the build phase, it'd be good to pull it out as a static.